zipkin/src/main/java/zipkin/internal/v2/Call.java

```diff
@@ -47,6 +49,37 @@
  * @param <V> the success type, only null when {@code V} is {@linkplain Void}.
  */
 public abstract class Call<V> implements Cloneable {
+  static final Call CALL_EMPTY_LIST = Call.create(Collections.emptyList());
+
+  @SuppressWarnings("unchecked")
+  public static <T> Call<List<T>> emptyList() {
+    return (Call<List<T>>) CALL_EMPTY_LIST.clone();
```

Comment:
I believe it would be the same exact logic if you use `Call.create(Collections.emptyList())` here instead of `clone`, but would let you avoid having the extra static constant.